src/shared/src/ata.ts

```diff
@@ -88,11 +88,11 @@ export const AtaChaptersTitle = {
     116: 'Flight Simulator Cuing System',
 };
 
-// temporary probably
 export const AtaChaptersDescription = Object.freeze({
     24: 'All things related to the electrical system. The electrical system supplies power from the engines, APU, batteries, or emergency generator to all cockpit instruments.',
     29: 'The hydraulic system connects to the flight controls, flaps and landing gear to provide pressure to these surfaces. Failing these can cause loss of control over some flight surfaces.',
     31: 'The cockpit displays give critical flight information to the pilots. In a failure where displays are lost, the pilots must deal with a lack of flight data given to them.',
+    34: 'The navigation systems provide data about the position, speed, heading, and altitude of the aircraft. Failures in a system such as the ADIRS can return erroneous data to the pilots',
```

Comment:
```suggestion
    34: 'The navigation systems provide data about the position, speed, heading, and altitude of the aircraft. Failures in a system such as the ADIRS can return erroneous data to the pilots.',
```

Also most of the time, the erroneous data is filtered out. I would just say loss of information